snapshot/generate.go

```diff
@@ -657,7 +654,14 @@ func (dl *diskLayer) generate(stats *generatorStats) {
 		}
 		// If the iterated account is the contract, create a further loop to
 		// verify or regenerate the contract storage.
-		if acc.Root == emptyRoot {
+		contractAcc, ok := acc.(*account.SmartContractAccount)
+		if !ok {
+			accMarker = nil
+			return nil
+		}
```

Comment:
Don't we need to call `wipeKeyRange` for EOA?

Reply:
@aidan-kwon Can you check this again?